src/components/shared/ContextMenu.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import React, { PureComponent } from 'react';
+import ReactDOM from 'react-dom';
+import { ContextMenu } from 'react-contextmenu';
+
+export default class MyContextMenu extends PureComponent<{}> {
```

Comment:
I think we can infer the props of ContextMenu like this:
```js
type Props = React.ElementConfig<typeof ContextMenu>;
```
Then you can use this infered `Props` for the parameter of `PureComponent` here, and we get a better type safety :)

(or at least we'll get a better type safety once `react-contextmenu` gets a lib definition... oh my).

Reply:
So after doing this, because react-contextmenu specifies `children: PropTypes.node.isRequired`, it results in some console.errors when no children are given. In order to satisfy react-contextmenu, I thought the correct thing to do would be to default to a single child when no children are given (which is what CallNodeContextMenu.js does). All the tests still pass and CallNodeContextMenu's render logic can also be simplified.